src/app/views/request/confirmation/confirmation.component.css

```diff
@@ -1,5 +1,5 @@
-div.content{
-    padding-top: 60px;
+div.content {
+    padding-top: 8rem;
```

Comment:
Use em for fonts, % for width margins. Top/bottom spacing should always use fixed units "px, pc, ect..." to avoid responsive errors.

Reply:
Not necessarily. Every device got different heights, 60px on one device might seem to fit one device, but other device's who doesn't have the same height, it will feel crammed or too large. 

I would actually suggest using vh = view height instead.

Reply:
The thing is, the height in that .content is due to the header, that always has the same height(doesn't matter the devices height)). Is not to set the content in the middle. If you use percentage for top-margins/top-positioning, you can expect to write rules for every resolution.

Reply:
Then we can set the height of the main content to be: "height: calc(100vh - header-height)"

Reply:
So the main content never goes above the device's height, and causes a scroll that shouldn't be there.

Reply:
With height 100vh and padding 60px it never will (if it is, we need to add box-sizing: border-box)